src/posh-winfetch.ps1

```diff
@@ -87,7 +101,7 @@ if ($help) {
 # ===== GENERATE CONFIGURATION =====
 if ($genconf.IsPresent) {
     if ((Get-Item -Path $config).Length -gt 0) {
-        write-host 'ERROR: configuration file already exists!' -f red
+        Write-Output 'ERROR: configuration file already exists!' -f red
```

Comment:
No. Errors need to be output with `write-host`.

Reason: in the event that the output of Winfetch gets redirected (say, to a file to be `cat`ed later), these errors won't be seen because they were written to output, not to the console (`write-host` writes to the console, `write-output` writes to output.).

Reply:
I made that change since it was recommended by the PSGalleryLinter `Invoke-ScriptAnalyzer`